playground/ProxylessEndToEnd/ProxylessEndToEnd.AppHost/Program.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddProject<Projects.ProxylessEndToEnd_ApiService>("api")
+    .WithEndpoint("http", ea =>
+    {
+        ea.UriScheme = "http";
+        ea.Port = 12345;
+        ea.IsProxied = false;
```

Comment:
Is this the only way to use it? Can you set it using the other WithEndpoint overloads?

Reply:
Can we add more the optional parameter to the base overload?